release-1.8/release-notes-draft.md

```diff
@@ -213,6 +213,18 @@ This section contains a list of known issues reported in Kubernetes 1.8 release.
 
 For more information, see [#1996](https://github.com/kubernetes/minikube/issues/1996).
 
+* Audit logs might impact the API server performance and the latency of large request and response calls. The issue is observed under the following conditions: `AdvancedAuditing` feature gate is enabled (`--feature-gates=AdvancedAuditing`), audit logging uses the log backend in JSON format, and the audit policy records large API calls at request and response level.
```

Comment:
Sorry, the actual flag setting should be `--feature-gate=AdvancedAuditing=true`. However, I would just leave that out and mention that it's on by default, although that is also stated above.